2024/1/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+D=$(dirname $(realpath $0))
+
+echo "-- Day 1 --"
+printf "%-10s %-15s %-10s %-10s %-10s %-10s \n" "language" "author" "compile" "total" "average" "status"
+$D/../../lang/node.sh                    "$D/solutions/node/stelar7/main.mjs"              "$D/io/*"
+$D/../../lang/python3.sh                 "$D/solutions/python/ayato/main.py"               "$D/io/*"
```

Comment:
Looks like wrong folder?

```suggestion
$D/../../lang/python3.sh                 "$D/solutions/python/renja-g/main.py"               "$D/io/*"
```